### DISCUSSION TITLE
integrate huggingface inference endpoint with flowise issue🆘

### DISCUSSION THREAD

**Discussion by For-Pubilc-uses** [FlowiseAI/Flowise#2042] [Q&A]
I am trying to integrate model: mistralai/Mixtral-8x7B-Instruct-v0.1 from hugging face which I have deployed as an inference endpoint already, and I got a URL that i can put into flowise [Add Endpoint URL](https://i.stack.imgur.com/hmwvO.png)

but it just doesn't work as expected, when i try to run the model, it shows that [Error](https://i.stack.imgur.com/evbCz.png),so I just set the max_token to 250, now I can generate the text but I still the model works randomly with no expected output [Model Output](https://i.stack.imgur.com/Nf05P.png)

it is supposed to be a translator that translates everything into English, this is how the model will behave when I used free inference API from hugging face: [Behaved Correctly](https://i.stack.imgur.com/ZwCqb.png)

there's a limit for free inference API, that's why I paid for the inference endpoint, but it just behaved so differently unlike the free inference api, so confused, please help me out if anyone knows, thanks so much!

**Comment by HenryHengZJ** [comment]
which chain are you using? here's a quick video about using HF models on FW - https://youtu.be/rGehI5PNP2o?si=v3t9qo2Fa8jy6nc5
